pages/linux/quotacheck.md

```diff
@@ -0,0 +1,28 @@
+# quotacheck
+
+> Scan a filesystem for disk usage; create, check and repair quota files.
+> Should only be run by the super-user.
+
+- Check all mounted non-NFS filesystems:
```

Comment:
while running quotacheck? How long does the scan last, and what kind of changes are you talking about? An example would help me understand this better :)